Messaging.md

```diff
@@ -336,11 +360,14 @@ Witnesses should be sorted by the `input_serial_id` sent in `funding_input` defi
 This type contains information about an event on which a contract is based.
 Two types of events are described, see [the oracle specification](./Oracle.md#event-descriptor) for more details.
 
-#### Version 0 `enum_event_descriptor`
+**For backward compatibility reasons, this type is currently serialized as a TLV. It is expected to be changed to the new serialization format in a near future.**
+
+#### `enum_event_descriptor`
 
-1. type: 55302 (`enum_event_descriptor_v0`)
-2. data:
-   * [`u16`:`num_outcomes`]
+1. implements: `event_descriptor`
+1. type: 55302
+1. data:
+   * [`bigsize`:`num_outcomes`]
```

Comment:
This also can break compatability (nested inside of oracle announcement). This is an issue with the existing test vector `enum_single_oracle_test.json`. 

When trying to read your `serialized` string, and then write it back and compare the our serialized string vs your serialized string it seems that the length of the oracle announcement is slightly different than what we get. I believe this is because `BigSize` is not deterministic with it's sizing, while `uint16` is (always 2 bytes). 

Here is the branch to reproduce: 

https://github.com/Christewart/bitcoin-s-core/commit/9e2123dbbca17f34f382d72413b1440e4d5178bc

and you can run this command inside of `sbt` to see the results

>coreTestJVM/testOnly *ContractInfoV0Test* -- -z  "conform to the enum single test vector"

If you look carefully at the image below, you see it's the `length` of the TLV that seems to be off by a few bytes.

![Screenshot from 2021-11-10 09-52-51](https://user-images.githubusercontent.com/3514957/141146474-a095022c-7b85-4926-8b37-997e255e4f53.png)

Reply:
Thanks I had forgotten to revert that part, I updated the test vectors I hope this time it's ok!